src/riak_kv_wm_aaefold.erl

```diff
@@ -26,6 +26,8 @@
 %% GET /cachedtrees/nvals/NVal/keysclocks?filter
 %% GET /rangetrees/[types/Type/]buckets/Bucket/trees/Size?filter
 %% GET /rangetrees/[types/Type/]buckets/Bucket/keysclocks?filter
+%% GET /rangerepl/[types/Type/]bucket/Bucket?filter
```

Comment:
It is kind of convention in REST land to use plural here `buckets` instead of `bucket` as in the other endpoints.

Reply:
In fact, if the Bucket is a bucket identifier, then the query `?filter` is a bit weird.

Reply:
The bucket/buckets is a typo in the comments - buckets is actually what is being used here.  Will correct.